python/servo/testing_commands.py

```diff
@@ -235,8 +234,6 @@ def test_unit(
             except KeyError:
                 pass
 
-        packages.discard("stylo")
-
```

Comment:
I was interested about why this `discard` call was added. It dates back to https://github.com/servo/servo/commit/1d497828a76ae0739463ef6a196fa4b98b06173c and back then you were supposed to run `./mach test-stylo`, not `./mach test-unit -p stylo`.